webpack.prod.js

```diff
@@ -10,5 +11,10 @@ module.exports = merge.smart(common, {
 			hashFuncNames: ['sha384'],
 			enabled: true,
 		}),
+		new PacktrackerPlugin({
+			project_token: '11bc4903-960f-4a31-b6aa-65e13e351fa2',
```

Comment:
Is it cool to leave the project_token in there? Not sure if that's secret.

Reply:
Nobody could really do anything bad with it but I might as well just put it in env vars.